test/e2e/utils_test.go

```diff
@@ -313,13 +315,65 @@ func checkInternalMemberClusterExists(name, namespace string) {
 func checkMemberClusterNamespaceIsDeleted(name string) {
 	Eventually(func(g Gomega) error {
 		var ns corev1.Namespace
-		if err := hubClient.Get(ctx, types.NamespacedName{Name: name}, &ns); !errors.IsNotFound(err) {
+		if err := hubClient.Get(ctx, types.NamespacedName{Name: name}, &ns); !k8serrors.IsNotFound(err) {
 			return fmt.Errorf("member cluster namespace %s still exists or an unexpected error occurred: %w", name, err)
 		}
 		return nil
 	}, eventuallyDuration, eventuallyInterval).Should(Succeed())
 }
 
+func setupNetworkingCRDs() {
+	var internalServiceExportCRD apiextensionsv1.CustomResourceDefinition
+	Expect(utils.GetObjectFromManifest("./manifests/internalserviceexport-crd.yaml", &internalServiceExportCRD)).Should(Succeed())
+	Expect(hubClient.Create(ctx, &internalServiceExportCRD)).Should(Succeed())
+	By("Networking CRDs are created")
+
+	Eventually(func(g Gomega) error {
+		err := hubClient.Get(ctx, types.NamespacedName{Name: "internalserviceexports.networking.fleet.azure.com"}, &internalServiceExportCRD)
+		if k8serrors.IsNotFound(err) {
+			return err
+		}
+		return nil
+	}, eventuallyDuration, eventuallyInterval).Should(Succeed())
+}
+
+func cleanupNetworkingCRDs() {
+	internalServiceExportCRD := apiextensionsv1.CustomResourceDefinition{
+		ObjectMeta: metav1.ObjectMeta{
+			Name: "internalserviceexports.networking.fleet.azure.com",
+		},
+	}
+	Expect(hubClient.Delete(ctx, &internalServiceExportCRD)).Should(Succeed())
+	By("Networking CRDs are deleted")
+}
+
+func createInternalServiceExport(name, namespace string) {
```

Comment:
Hi Arvind! This is just a nit, but this file is for utilities that incorporate complex logic; and this function seems to be just creating a resource. If you are just looking for resource getter, add one in `resources_test.go` and call the create method in the test code.